spec/features/collection/collection_spec.rb

```diff
@@ -286,6 +246,14 @@
       visit "/collections/#{collection_modify.id}"
     end
 
+    it "should not have edit and delete options" do
+      expect(page).to have_content(generic_file.title.first)
+
+      expect(page).to have_content("Test Item")
+      expect(page).to have_content("Download File")
+      expect(page).not_to have_content("Edit File")
```

Comment:
As the interface has changed, these two lines are no longer relevant (as they were from the action dropdown menu in the old collection landing page) @criedlberger I will make the change.